test/kubernetes_unittest.cc

```diff
@@ -13,6 +15,16 @@ class KubernetesTest : public ::testing::Test {
       throw(json::Exception) {
     return reader.GetNodeMetadata(node, collected_at, is_deleted);
   }
+
+  json::value ComputePodAssociations(const json::Object* pod,
```

Comment:
Just noticed — let's make `reader` the first argument, both here and in `UpdateOwnersCache`.

Reply:
Done